Makefile

```diff
@@ -15,7 +15,7 @@ docs_build:
 	docs/.local_build.sh
 
 docs_clean:
-	rm -r docs/_dist
+	rm -rf docs/_dist
```

Comment:
Make docs_clean idempotent, this way it won't error out if the dir doesn't exist